scripts/lib-franklin.js

```diff
@@ -735,15 +727,10 @@ export function decorateButtons(element) {
 }
 
 /**
- * Load LCP block and/or wait for LCP in default content.
+ * Wait for Image.
  */
-export async function waitForLCP(lcpBlocks) {
-  const block = document.querySelector('.block');
-  const hasLCPBlock = block && lcpBlocks.includes(block.dataset.blockName);
-  if (hasLCPBlock) await loadBlock(block);
-
-  document.body.style.display = null;
-  const lcpCandidate = document.querySelector('main img');
+export async function waitForImage(section) {
```

Comment:
I wonder if we should be more explicit and rename to `waitForFirstImage`, or alternatively allow specifying a custom selector:
```js
export async function waitForImage(section, selector = 'img') {
  const lcpCandidate = section.querySelector(selector);
  …
}

Reply:
agreed